.github/workflows/build_and_test.yml

```diff
@@ -77,8 +77,51 @@ jobs:
       run: echo "::error file=.github/workflows/build_and_test.yml::File not included in any filter" && false
       if: ${{ !contains(steps.filter.outputs.*, 'true') }}
 
-  rust:
-    name: Rust
+  rust-stable:
+    name: Rust stable
+
+    runs-on: ubuntu-latest
+
+    needs: changes
+
+    if: ${{ needs.changes.outputs.rust == 'true' }}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - run: sudo apt-get install gcc-multilib
+
+    - name: Install Rust
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable
+        components: rustfmt, clippy
```

Comment:
Actually, do we need the install at all? The [default GitHub Ubuntu image has a recent stable](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md). `rustup target add i686-unknown-linux-gnu` might be enough. (I went back and forth on whether the 32-bit stable testing was really necessary but why not.)

Reply:
At first try, it seemed to install `nightly` because of the `rust-toolchain` file (which is why I suggested to drop it in the first place). 

I guess at least `rustfmt` and `clippy` can be dropped though.

Reply:
`rustup +stable target add i686-unknown-linux-gnu`, then.